src/aks-preview/HISTORY.rst

```diff
@@ -27,6 +27,7 @@ Pending
 +++++++
 * Change the format for az aks machine commands to separate the ipv4, ipv6 columns
 * Deprecate the alias "-r" of parameter --source-resource-id in `az aks trustedaccess rolebinding create`
+* Add `az aks nodepool delete-machines` command.
```

Comment:
OK, but `1.0.0b3` has been released, please move your history note to `Pending` if you don't want to release a new version of aks-preview.

OR choose a version number like `1.0.0b6`, put your history note and move `* Vendor new SDK and bump API version to 2023-11-02-preview.` there, and update `VERSION`  in `setup.py`

Reply:
Thanks. Move it to pending section.